src/components/Capture/Face.js

```diff
@@ -87,28 +87,42 @@ class Face extends Component {
     // when we finally do get its value
     if (hasCamera === null) return
 
-    return (
-      useWebcam && hasCamera ?
-        requestedVariant === 'video' ?
+    if (hasCamera) {
+      if (requestedVariant === 'video') {
+        return (
           <Video
             {...cameraProps}
             onVideoCapture={ this.handleVideoCapture }
-          /> :
+          />
+        )
+      }
+
+      // FIXME: remove code dependency on useWebcam once PR #762 for UI tests refactor is merged into 'development' branch
+      //        (useWebcam is meant to be used to enable document autocapture feature that is still in beta)
+      if (this.props.useWebcam === true) {
+        return (
           <Selfie
             {...cameraProps}
             onCapture={ this.handleCapture }
             useMultipleSelfieCapture={ useMultipleSelfieCapture }
             snapshotInterval={ snapshotInterval }
-          /> :
-      this.props.uploadFallback ?
+          />
+        )
+      }
+    }
+
+    if (this.props.useWebcam === false || hasCamera === false || uploadFallback) {
```

Comment:
Why not using `!this.props.useWebcam` so it catches falsey values such as `undefined`?

Reply:
Yes, you're right!